.nvmrc

```diff
@@ -1 +1 @@
-16.14.0
\ No newline at end of file
+18.16.1
```

Comment:
This is not the latest version of Node 18 (it's on 18.19 now) but it is the latest version that has a `cypress/base` image.